policykit/policyengine/views.py

```diff
@@ -646,10 +646,10 @@ def role_action_users(request):
     action.community = user.community
     action.initiator = user
     action.role = CommunityRole.objects.filter(name=data['role'])[0]
-    action.save()
+    action.save(without_govern=True)
     action.users.set(CommunityUser.objects.filter(username=data['user']))
     action.ready = True
-    action.save()
+    action.save(treat_as_new=True)
```

Comment:
@amyxzhang @rickygv99 what's the desired behavior here? I'm not sure what should happen.

It looks like there is already a special case for `PolicykitAddUserRole` and `PolicykitRemoveUserRole` which override the "shouldCreate" function to use this `ready` flag, as a way to defer policy evaluation until the second time the model is saved... So that seems similar to what @hozzjss is trying to introduce here with "treat_as_new"

https://github.com/amyxzhang/policykit/blob/da19825490164e770b83c9339a3775ae02bd28d9/policykit/policyengine/models.py#L751-L752